crimethinc/website
# Add `lang=foo` url param to the tools controller

# How does this pull request make you feel (in animated GIF format)?

![alt text](https://media.giphy.com/media/JqDeI2yjpSRgdh35oe/giphy.gif)

# What are the relevant GitHub issues?

resolves `[link to github issue]`

# What does this pull request do?

This will allow you to filter the index of tools to a specific
language. `preferred_localization` still takes precedence with
deciding which translation to render, but the filter guarantees that
you will only be seeing tools that have as translation in your
filtered language

# How should this be manually tested?

- go to review app
- see if filter works as expected

# Questions for the pull request author (delete any that don't apply):
